lib/task-data/schemas/generate-tag.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$schema": "rackhd-task-schema.json",
+    "copyright": "Copyright 2016, EMC, Inc.",
+    "title": "Generate Tag Task",
+    "description": "The schema for generate-tag",
+    "describeJob": "Job.Catalog.GenerateTag",
+    "allOf": [
+        { "$ref": "common-task-options.json#/definitions/Options" },
+        {
+            "oneOf": [
```

Comment:
The sub-schemas within `oneOf` doesn't need to be so complex, unlike `anyOf`, `oneOf` is to assert it conforms to ONLY one sub-schema, it will fail if conforms to two or more sub-schemas. So it can be simplified to:
```json
"allof": [
   { "$ref": "common-task-options.json#/definitions/Options" },
   {
        "properties": {
            "nodeId": {
                "type": "string",
                "description": "An individual nodeId to process tags against"
            },
            "nodes": {
                "type": "array",
                "description": "An array of nodeIds to process tags against",
                "items": {
                    "type": "string"
                }
            }
        },
        "oneOf": [
            { "required": ["nodeId"] },
            { "required": ["nodes"] }
        ]
   }
]
```